src/platform/wayland.rs

```diff
@@ -73,9 +80,21 @@ impl EventLoopBuilderExtWayland for EventLoopBuilder {
 /// Additional methods on [`Window`] that are specific to Wayland.
 ///
 /// [`Window`]: crate::window::Window
-pub trait WindowExtWayland {}
+pub trait WindowExtWayland {
+    /// Returns `xdg_toplevel` of the window or null if the window is X11 window.
+    fn xdg_toplevel(&self) -> *mut c_void;
```

Comment:
Could you return an `Option<NonNull>` instead?

Reply:
Okay. Will do after next week since I don't have access to my computer.

Reply:
Done.